__testfixtures__/ember-qunit-codemod/subject.output.js

```diff
@@ -26,8 +26,10 @@ module('Unit | Model | Foo', function(hooks) {
     let subject = run(() => this.owner.lookup('service:store').createRecord('foo'));
   });
 
-  test('has another thing', function (assert) {
-    let subject = run(() => this.owner.lookup('service:store').createRecord('foo', { size: 'big' }));
+  ['big', 'small'].forEach(function (size) {
+    test('has another thing', function (assert) {
+      let subject = this.owner.factoryFor('component:foo-bar').create({ size });
```

Comment:
Should this be `let subject = run(() => this.owner.lookup('service:store').createRecord('foo', { size }));` ?

Reply:
sorry I keep moving the code snippet to the wrong place... the lines with similar names make it hard to grok where I am.
I'll move it again